_data/hosting_providers.json

```
@@ -1619,6 +1619,16 @@
     "reviewed": "",
     "note": ""
   },
+  {
+    "name": "Seravo",
+    "link": "https://seravo.com",
+    "category": "full",
+    "tutorial": "https://help.seravo.com/article/24-how-do-i-enable-the-https-on-our-website",
+    "announcement": "https://seravo.com/blog/https-is-not-optional/",
+    "plan": "",
+    "reviewed": "2020.11.09",
+    "note": "HTTPS is mandatory for sites at Seravo. All domains have automatically Let's Encrypt certificates and they are enabled by default."
```

The note field is for things like "this language is in German".
```suggestion
    "note": ""
```

---

Removed.